### DISCUSSION TITLE
Outline / Forge / SSH

### DISCUSSION THREAD

**Discussion by adrianbulldozer** [OutlineFoundation/outline-sdk#75] [Q&A]
I really need help with figuring out how to get everything to work with outline. I recently got a new computer and totally forgot how I set everything up on my old one as this was ages ago. I’m trying to SSH to one of the servers on Laravel Forge (Digital Ocean) but I can’t and I don’t know why.
Can you please help me with this? I understand you probably have follow-up questions and I will do my best to provide an answer.

Chreers

**Comment by maddyhof** [comment]
Thank you for reaching out! This Discussion section is still new but is focused on developers using Outline SDK. For support with your question, I recommend you submit the contact form at [support.getoutline.org](https://support.getoutline.org/) and include the GitHub Discussion number. 

(If by any chance you started on that form but were routed here, make sure to select "I am using the Outline Client application on my mobile or desktop device, or I want to install it" since your question is about Outline Client. I apologize for any back-and-forth as we adjust to having the SDK discussion board and an updated contact form.)
